tedana/metrics/collect.py

```diff
@@ -361,9 +360,11 @@ def generate_metrics(
                     echo=(i_echo + 1)
                 )
 
-    # Build new comptable, re-ordered like previous tedana versions
-    previous_order = (
-        "kappa", "rho", "variance explained",
+    # Reorder component table columns based on previous tedana versions
+    # NOTE: Some new columns will be calculated and columns may be reordered during
+    # component selection
+    preferred_order = (
```

Comment:
I simplified this section, but the thing I want to note is that the component table will be re-sorted in the selection stage, so I think we should move this part to the selection functions instead. I don't want to do that in this PR though.

Reply:
I think that's fine, this was a hack to make it straightforward to compare "old" and "new" metrics after the PR. We can have a discussion about it in a different issue/PR.